create_issue.py

```
@@ -0,0 +1,253 @@
+    resp = requests.post(url, headers=HEADERS, json=data, timeout=30)
+    resp.raise_for_status()
+    
+    return resp.json()
+
+def main():
+    print("=" * 70)
+    print("🎯 CRIADOR DE ISSUES COM IA")
+    print("=" * 70)
+    
+    # Solicitar parâmetros customizados (ou usar padrões)
+    print(f"\n📋 Configurações atuais:")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)